string name
uint8 state
float32 velocity
float32 acceleration